router/pkg/config/config.schema.json

```diff
@@ -2595,6 +2595,10 @@
                 "type": "string",
                 "description": "The name of the response header from which to extract the value. The value is only extracted for subgraph access logs"
               },
+              "expression": {
```

Comment:
@StarpTech This config is still shared between subgraphs and router (i.e. :- there wont be a schema validation error if you specify "expression" in a subgraph config, but it will be a no-op) , I was trying out using using allOf to get merging two subschemas to work, but didn't make too much progress yet (I didn't want to duplicate the schema config for router and subgraph of course for just one field)

If its good with you, I would prefer to open this as a separate PR?